setup-ovs.sh

```diff
@@ -36,6 +36,9 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
   echo -e "default dev: $default_device ($profile_name)\nsecondary dev: $secondary_device ($secondary_profile_name)"
   
   mac=$(sudo nmcli -g GENERAL.HWADDR dev show $default_device | sed -e 's/\\//g')
+
+  # delete old bridge if it exists
+  ovs-vsctl --if-exists del-br brcnv
```

Comment:
forgive the maybe stupid question, but if there is already a `brcnv` configured won't we skip the entire script on [line 10](https://github.com/RHsyseng/rhcos-slb/pull/18/commits/af2056f7dcd330a0a54acd61491e0542ff053b5f#diff-0a9487990caae3642584ba9967e53397d40b4b458900cd045dd87e4cef11198dR10)?
```
if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
```

Reply:
This comes from: https://bugzilla.redhat.com/show_bug.cgi?id=1952229

Basically due to some changes in CoreOS, the NetworkManager config gets lost, but the interfaces are still in the OvS DB

Reply:
ah I understand.

Reply:
In case its lost do we want to keep this line then? 
I mean if for some reason something happened and the line exits do we want to keep the configuration or build it again to make sure we are building it right?

Reply:
NetworkManager config is deleted, yes. However, OVS has its own persistent database to store all the configuration, this one is not flushed upon reboot. IIUIC that's the reason to explicitly clean up here.